release-notes/README.md

```diff
@@ -4,6 +4,17 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## December 15, 2021
+
+### Android Core 1.9.1
+
+* Fixed an issue which caused the duplicate query parameters in the deep link to be removed. 
```

Comment:
Let's change to:
Fixed an issue that was causing duplicate question parameters in a deep link to be removed.